lib/ddtrace/chunker.rb

```diff
@@ -22,7 +22,7 @@ def chunk_by_size(list, max_chunk_size)
         chunk_agg += size
         if chunk_agg > max_chunk_size
           # Can't fit element in current chunk
-          # TODO Datadog::Debug::HealthMetrics.increment('tracer.chunker.chunked')
+          Datadog.health_metrics.transport_chunked(1, tags: ["max_size:#{max_chunk_size}"])
```

Comment:
Not sure we need to emit a metric each time a chunk occurs. Is it possible to do it outside this function, say perhaps where `chunk_by_size` or `encode_in_chunks` is called, and emit one metric with the number of chunks in aggregate? Would give the insight necessary into how many chunks are produced without having a 1:1 number of metrics.

Reply:
Moved to bulk reporting now 👍